logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/AvsClient.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.logic.data.call
+
+data class AvsClient(
+    val userId: String,
+    val clientId: String
+)
+
+data class AvsClientList(
+    val clients: List<AvsClient>
+) {
+    fun toJsonString(): String {
+        val start = "{"
+        var startClients = "clients:["
+
+        clients.forEachIndexed { index, avsClient ->
+            startClients += if (index > 0) "," else ""
+            startClients += "{userid: ${avsClient.userId},clientid: ${avsClient.clientId}}"
+        }
+
+        val endClients = "]"
+        val end = "}"
+
+        return "$start$startClients$endClients$end"
+    }
```

Comment:
What this function does is basically: `data class -> json object -> string`.

I haven't found an easier and better way, if you know how please let me know 🙏🏻

Reply:
I think if you add a `@Serializable` annotation to the classes `AvsClient` and `AvsClientList` you could just do the conversion by:

`Json { isLenient = true }.encodeToString(AvsClientList.serializer(), yourAvsClienListInstance).toString()`

I did a quick test with another class and works, if you want to try it =)